packages/framework/fluid-framework/src/test/tsconfig.json

```diff
@@ -1,19 +0,0 @@
-{
-    "extends": "@fluidframework/build-common/ts-common-config.json",
-    "compilerOptions": {
-        "rootDir": "./",
-        "outDir": "../../dist/test",
-        "declaration": false,
-        "declarationMap": false,
-        "skipLibCheck": true,
```

Comment:
was the skipLibCheck stuff fixed?

Reply:
oh nvm the typetest stuff is disabled here ok

Reply:
this package doesn't have type tests right now, as there are problems with const enums coming from the merge tree package. this is something that need to be fix #8142. So this was tsconfig for nothing that was just causing problems